src/sentry/static/sentry/app/components/charts/components/axisPointer.jsx

```diff
@@ -0,0 +1,5 @@
+export default function AxisPointer({...props} = {}) {
```

Comment:
Doesn't look like we need to load a module for this (from https://github.com/hustcc/echarts-for-react)